src/DataStructureGroupParser.h

```diff
@@ -0,0 +1,154 @@
+//
+//  DataStructureGroupParser.h
```

Comment:
I have actually did `git mv`, but when the changes go above a certain %, git treats it as normal add and delete.